### PR TITLE
fix: Removing ember-scroll-to addon

### LINKED ISSUE #2544: When scrolling to a target , The target name should get highlighted .
**Describe the bug**

In the sidebar when user clicks on Scrollable objects and button doesn't highlights their heading

**To Reproduce**

Steps to reproduce the behaviour:

1. View any event
2. Click on *Scrollable object like Tickets or Getting Here* .
3. They dont get highlighted and info is highlighted rather.

**Expected behaviour**

They should be highlighted as 'info'
**Screenshot**

image

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Browser: Chrome
 - Version: 73.0.3

**Additional context**
I am on it

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #2544 

#### Changes proposed in this pull request:

Preview link - [https://deploy-preview-3852--open-event-fe.netlify.com/e/93efcaec](https://deploy-preview-3852--open-event-fe.netlify.com/e/93efcaec)

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by snitin315** [comment]
> [#3562 (comment)](https://github.com/fossasia/open-event-frontend/pull/3562#issuecomment-570115491)

@kushthedude 
I have gone through `scroll-to docs` , it doesn't provide much functionality.

Screenshot from 2020-01-27 20-11-57


So we have to write actions manually if we want to use `scroll-to`

**Comment by kushthedude** [comment]
Can you check if this is the only place where we are using scroll to?

**Comment by snitin315** [comment]
[Yes It's the only place](https://github.com/fossasia/open-event-frontend/search?q=scroll-to&unscoped_q=scroll-to) .

**Comment by kushthedude** [comment]
Since we are only using the scroll property from the following add-on . 
Do you think you would be able to write custom action for scrolling by taking reference from here https://github.com/akshay-kr/ember-scroll-to-target/blob/master/addon/components/scroll-to.js ? 
Scrolling is quite a simple property we should eliminate the add-on, if we can .

**Comment by snitin315** [comment]
> Do you think you would be able to write custom action for scrolling by taking reference from here https://github.com/akshay-kr/ember-scroll-to-target/blob/master/addon/components/scroll-to.js ?

Yes , I can give it a try.

**Comment by snitin315** [comment]
Preview link - [https://deploy-preview-3852--open-event-fe.netlify.com/e/93efcaec](https://deploy-preview-3852--open-event-fe.netlify.com/e/93efcaec)

**Comment by kushthedude** [comment]
@snitin315 Update it

File: app/components/public/side-menu.js [hunk 1]
```diff
@@ -9,6 +10,22 @@ export default Component.extend({
...
+          document.querySelector(this.getAttribute('href')).scrollIntoView({
+            behavior: 'smooth'
+          });
+          $('.scroll').each(function() {
+            $(this).removeClass('active');
+          });
+          $(this).addClass('active');
+        });
+      });
+    }
```
Comment (iamareebjamal) [inline app/components/public/side-menu.js]: We are moving away from jquery, not towards it

**Comment by snitin315** [comment]
@iamareebjamal removed jQuery :+1:

**Pull request merged**